### PR TITLE
Set up a Video Indexer enabled by Arc environment.

### PR DESCRIPTION
**Pull request by shemers**
**Is your feature request related to a new Jumpstart scenario you would like to contribute?**

Yes, a new scenario.

**Is your feature request related to a problem? Please describe.** No

**Describe the solution you'd like.**
How to set up a Video Indexer enabled by Arc environment.

**Describe alternatives you've considered.**
N\A

**Additional context.**
N\A

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+{
+    "speechCognitiveServicesPrimaryKey": "<key>",
+    "speechCognitiveServicesSecondaryKey": "<key>",
+    "translatorCognitiveServicesPrimaryKey": "<key>",
+    "translatorCognitiveServicesSecondaryKey": "<key>",
+    "speechCognitiveServicesEndpoint": "<uri>",
+    "translatorCognitiveServicesEndpoint": "<uri>"
+}
+```
+
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md:123]: show screenshot showing success

File: docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                            --config "speech.resource.requests.mem=2Gi" \
+                            --config "speech.resource.limits.cpu=1" \
+                            --config "speech.resource.limits.mem=4Gi" \
+                            --config "videoIndexer.webapi.resources.requests.mem=4Gi"\
+                            --config "videoIndexer.webapi.resources.limits.mem=8Gi"\
+                            --config "videoIndexer.webapi.resources.limits.cpu=1"\
+                            --config "storage.storageClass=azurefile-csi" 
+
+```
+
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md:202]: show screenshot with success. Also be sure to show an example that uses every parameter in an example basic configuration. Dont assume user will know which values to use for any parameter and explicitly show what will work with the recommended setup presented in the guide.

File: docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+### Step 4 - Verify Deployment
+
+```shell
+kubectl get pods -n video-indexer
+```
+
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md:208]: show screenshot with successful kubectl output

File: docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+                            --resource-group ${connectedClusterRg} \
+                            --cluster-type connectedClusters \
+                            --release-train ${releaseTrain}  \
+                            --version ${version} \
+                            --config "speech.resource.requests.cpu=500m"
+
+                            
+```
+
+> **NOTE:** You must specify the cluster name and resurce group name in order to update the extension.
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md:225]: NOTE styling

File: docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+                            --cluster-name ${connectedClusterName} \
+                            --resource-group ${connectedClusterRg} \
+                            --cluster-type connectedClusters \
+                            --release-train ${releaseTrain}  \
+                            --version ${version} \
+                            --config "speech.resource.requests.cpu=500m"
+
+                            
+```
+
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md:224]: show screenshot with successful output

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
Thanks for your PR.

Several things to look at:

* See my comments about styling for NOTE and use of "TIP". 
* Need to show screenshots to help user visualize what to expect
* Need to add a section at the end that points to how to clean up the environment. You can do this by linking to one of the existing K8s scenarios cleanup section, or you can do it by explaining to user to delete the RG used if using AKS. 
* For the section where user has to configure some parameters to pass to the extension, be very explicit and show a working command with the exact command to run. You can see examples of this in other scenarios or in ArcBox docs.
* Consider making the scenario anchored around a specific K8s basic scenario rather than giving user option of using AKS or bring their own K8s. I think it will be clearer if the user is following one guide to setup a base K8s distro, and then they use this guide which refers back to the guide they just followed.

**Comment by likamrat** [comment]
Closing this for now as @dkirby-ms working on a new VI scenario

**Pull request closed without merging**